src/components/billingRecord/IFXBillingRecordDetail.vue

```diff
@@ -193,19 +207,22 @@ export default {
       return item.account ? this.$api.organization.parseSlug(item.account.organization).name : ''
     },
   },
-  watch: {},
+  watch: {}
 }
 </script>
 <template>
   <!-- eslint-disable vue/valid-v-slot -->
   <v-container v-if="!isLoading">
     <IFXPageHeader>
-      <template #title>Billing Record {{ item.id }}</template>
+      <template #title>
+        Billing Record {{ item.id }}
+      </template>
       <template #actions>
         <IFXButton v-if="canEdit" btnType="edit" xSmall @action="openEditDialog()" />
       </template>
     </IFXPageHeader>
     <v-container px-5 py-0>
+      <v-progress-linear indeterminate v-if="updating"></v-progress-linear>
```

Comment:
Added a progress bar for updates.  In my dev environment, the update is a little slow when coming from fiine making it look like it had failed silently.  Almost certainly won't appear in facility mode.